protocol/synthetix/subgraph/package.json

```diff
@@ -11,12 +11,12 @@
     "auth": "graph auth --product hosted-service $V3_GRAPH_KEY",
     "deploy:mainnet": "graph deploy snx-v3/mainnet subgraph.mainnet.yaml --output-dir ./build/mainnet --product hosted-service",
     "alchemy:mainnet": "graph deploy synthetix-mainnet subgraph.mainnet.yaml --output-dir ./build/mainnet --version-label v2 --node https://subgraphs.alchemy.com/api/subgraphs/deploy --deploy-key $SATSUMA_KEY --ipfs https://ipfs.satsuma.xyz",
-    "deploy:goerli": "graph deploy snx-v3/goerli subgraph.goerli.yaml --output-dir ./build/goerli --product hosted-service",
+    "deploy:sepolia": "graph deploy snx-v3/sepolia subgraph.sepolia.yaml --output-dir ./build/sepolia --product hosted-service",
```

Comment:
we no longer use graph afaik and this should become alchemy?

Reply:
yeah I added also a alchemy script. We could also remove entirely the graph scripts but its also not like its blocking us